kolibri/plugins/user_profile/assets/src/views/ChangeFacility/MergeUsernameExists.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
+    <p data-test="line1">
+      {{ mergeAccountInfoLine1 }}
+    </p>
+    <p data-test="line2">
+      {{ mergeAccountInfoLine2 }}
+    </p>
+
+    <BottomAppBar>
+      <slot name="buttons">
+        <KButtonGroup>
+          <KButton
+            :primary="false"
+            :text="coreString('backAction')"
+            appearance="flat-button"
+            data-test="backButton"
+            @click="sendBack"
+          />
+          <KButton
+            :primary="true"
+            :text="coreString('continueAction')"
+            data-test="continueButton"
+            @click="handleContinue"
+          />
+        </KButtonGroup>
+      </slot>
+    </BottomAppBar>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import { computed, inject } from 'kolibri.lib.vueCompositionApi';
+  import get from 'lodash/get';
+
+  export default {
+    name: 'MergeUsernameExists',
+    metaInfo() {
+      return {
+        title: this.$tr('documentTitle'),
+      };
+    },
+    components: { BottomAppBar },
+
+    mixins: [commonCoreStrings],
+
+    setup(_, context) {
+      const changeFacilityService = inject('changeFacilityService');
+      const state = inject('state');
+      const store =
+        context.root.$store !== undefined ? context.root.$store : context.root.$children[0].$store;
+      const session = store.getters['session'];
+      const username = computed(() => session.username);
+      const mergeAccountInfoLine1 = computed({
+        get() {
+          return this.$tr('confirmMergeInfoLine1', {
+            target_facility: get(state, 'value.targetFacility.name', ''),
```

Comment:
It would be good to keep the variable, property, etc names consistent with the known Javascript conventions, unless there is a reason why this should be so for this case. The same would apply to [Line:73](https://github.com/learningequality/kolibri/pull/9687/files#diff-b6d35063aff073cfabb167aec1a23a7fc14dc458fd3e26b5204784eb8eb1d9f4R73)

Reply:
it was not actually a JS variable but a translation variable, but happy to change it.